.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
```

Comment:
Ah, that makes sense. Yeah, it couldn't hurt to add iojs. I haven't used iojs yet and I don't really know what specifically led people to fork node.js into a new project.